ql/src/java/org/apache/hadoop/hive/ql/metadata/Hive.java

```diff
@@ -2620,18 +2622,18 @@ public static void listFilesInsideAcidDirectory(Path acidDir, FileSystem srcFs,
     }
   }
 
-  private void listFilesCreatedByQuery(Path loadPath, long writeId, int stmtId, boolean isInsertOverwrite,
-      List<Path> newFiles) throws HiveException {
+  private List<FileStatus> listFilesCreatedByQuery(Path loadPath, long writeId, int stmtId) throws HiveException {
```

Comment:
insertOverwrite flag was never used? :)

Reply:
It was used in the original insertOverWrite patch, but later it was removed.